src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -308,18 +309,19 @@ private static ConversionKind ClassifyImplicitConstantExpressionConversion(Bound
                 if (nt.OriginalDefinition.GetSpecialTypeSafe() == SpecialType.System_Nullable_T &&
                     HasImplicitConstantExpressionConversion(source, nt.TypeArgumentsNoUseSiteDiagnostics[0]))
                 {
-                    return ConversionKind.ImplicitNullable;
+                    return new Conversion(ConversionKind.ImplicitNullable, Conversion.ImplicitConstantUnderlying);
                 }
             }
 
-            return ConversionKind.NoConversion;
+            return Conversion.NoConversion;
         }
 
-        private ConversionKind ClassifyImplicitTupleLiteralConversion(BoundExpression source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        private Conversion ClassifyImplicitTupleLiteralConversion(BoundTupleLiteral source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
```

Comment:
If I rename it, it would clash with "Get". 
Get just gets a tuple conversion if exists. Classify may also return a nullable conversion whose underlying conversion is a tuple conversion.
"Get" is also consistent with other abstract methods in ConversionBase  - like GetInterpolatedStringConversion who require an access to a binder.